src/battle_script_commands.c

```diff
@@ -17000,3 +17006,34 @@ void BS_TryQuash(void)
     }
     gBattlescriptCurrInstr = cmd->nextInstr;
 }
+
+void BS_CopyFoesStatIncrease(void)
+{
+    NATIVE_ARGS(u8 battler, const u8 *jumpInstr);
+    u32 stat = 0;
+    u32 battler = GetBattlerForBattleScript(cmd->battler);
+
+    if (gQueuedStatBoosts[battler].stats == 0)
+    {
+        memset(gQueuedStatBoosts, 0, sizeof(gQueuedStatBoosts)); // erase all boosts so they are not stacked
+        gBattlescriptCurrInstr = cmd->jumpInstr;    // stats done, exit
+        return;
+    }
```

Comment:
Won't this break Opportunist and Mirror Herb that activate at the same time?

Reply:
yes, there was actually a bug regardless if I erased it or not. I tried to work around that by setting all the bits back that have a stat increase.